### PR TITLE
Changeling Morph

### PR DESCRIPTION
**Pull request by Geevies**
* Lings can now evolve the ability to release members of their internal hivemind as morphs at the cost of their arm. The ability itself costs two genome points
* Added morphs, creature capable of assuming the form of any object.
* Messages spoken in the internal hivemind will automatically be capitalized and punctuated. Also improved the colour for readability.
* Fixed the script error in the Changeling DNA buy menu, improved the look of it.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Chada-1** [comment]
This is amazing, thank you V much geeves!!

**Review by Chada-1 — APPROVED** [review]

File: code/game/gamemodes/changeling/implements/powers/hivemind.dm [hunk 1]
```diff
@@ -34,4 +34,48 @@
...
+
+	if(!length(mind.changeling.hivemind_members))
+		to_chat(src, SPAN_WARNING("We have no internal hivemind members to release!"))
+		return
+
+	var/chosen_player = input("Choose a hivemind member to release as a morph.", "Hivemind Morph") as null|anything in mind.changeling.hivemind_members
+	if(!chosen_player)
+		return
+
+	var/mob/living/carbon/human/H = src
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/changeling/implements/powers/hivemind.dm:62]: no, bad

File: code/modules/mob/living/simple_animal/hostile/morph.dm [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+	tameable = FALSE
+
+	status_flags = CANPUSH
+	pass_flags = PASSTABLE
+
+	maxHealth = 125
+	health = 125
+
+	melee_damage_lower = 12
+	melee_damage_upper = 16
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/hostile/morph.dm:33]: i'm 99% sure there's another melee damage variable that you need to set here, but i can't remember it, harm_intent_damage?

File: code/modules/mob/living/simple_animal/hostile/morph.dm [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+	var/static/list/blacklist_typecache = typecacheof(list(/obj/screen, /obj/singularity, /mob/living/simple_animal/hostile/morph, /obj/effect))
+
+/mob/living/simple_animal/hostile/morph/Initialize()
+	. = ..()
+	verbs += /mob/living/proc/ventcrawl
+
+/mob/living/simple_animal/hostile/morph/examine(mob/user)
+	if(morphed)
+		. = form.examine(user)
+		if(get_dist(src, user) > 3)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/hostile/morph.dm:63]: 3 may be a bit too much here, I'd do 2

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/morph.dm (hunk 2 above)
Comment (Geevies) [reply code/modules/mob/living/simple_animal/hostile/morph.dm:33]: that affects how much damage it takes, not dishes out, iirc

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**